tutorials/index.md

```diff
@@ -64,6 +64,278 @@ If you are familiar with fMRI analysis, Python and machine learning:
 <div class="block-narrow block-secondary app-about-block" id="preview" markdown="1">
 <div class="container" markdown="1">
 ## Getting started<a name="getting-started"></a>
-Detailed installation instructions coming soon!
+Some guidance for how to access / install the tutorials:
+- **I want to try the tutorials for myself**: Use Google Colaboratory, Docker, or Conda.
+- **I want to set up for multiple users or use BrainIAK for research (i.e., not just running tutorials)**: Use Conda
+</div>
+</div>
+
+<div class="block-narrow block-inverse block-secondary app-code-block" id="about" style="line-height: 1.55;">
+<div class="container" markdown="1">
+## Detailed installation instructions
+
+<ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
+  <li class="nav-item">
+    <a class="nav-link active" id="pills-docker-macos-tab" data-toggle="pill" data-target="#pills-docker-macos" role="tab" aria-controls="pills-docker-macos" aria-selected="true">Docker (MacOS)</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="pills-docker-windows-tab" data-toggle="pill" data-target="#pills-docker-windows" role="tab" aria-controls="pills-docker-windows" aria-selected="false">Docker (Windows)</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="pills-conda-macos-tab" data-toggle="pill" data-target="#pills-conda-macos" role="tab" aria-controls="pills-conda-macos" aria-selected="false">Conda (MacOS)</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="pills-server-scotty-tab" data-toggle="pill" data-target="#pills-server-scotty" role="tab" aria-controls="pills-server-scotty" aria-selected="false">Server</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="pills-cluster-spock-tab" data-toggle="pill" data-target="#pills-cluster-spock" role="tab" aria-controls="pills-cluster-spock" aria-selected="false">Cluster</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="pills-google-colab-tab" data-toggle="pill" data-target="#pills-google-colab" role="tab" aria-controls="pills-google-colab" aria-selected="false">Google Colaboratory</a>
```

Comment:
Could you order it so that Colab is the first tab, and users can see the Colab instructions open, rather than having them click and then see the instructions?